pkgs/tools/misc/kisslicer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ fetchurl
+, libX11
+, mesa
+, makeWrapper
+, stdenv
+, unzip
+, inidir ? "\\\$HOME/.config/kisslicer"
+}:
+
+let
+  name = "kisslicer";
+  version = "1.6.2";
+
+  libPath = stdenv.lib.makeLibraryPath [
+    mesa
+    stdenv.cc.cc
+    libX11
+  ];
+
+in stdenv.mkDerivation rec {
+  inherit version name;
+
+  src = fetchurl {
+    url = "http://www.kisslicer.com/uploads/1/5/3/8/15381852/kisslicer_linux64_1.6.2_release.zip";
+    sha256 = "1aag3cywjz06s67031zpdxj8qzgalybyrg52axx6y6a40jfhxdn4";
+  };
+  unpackPhase = ''
+    mkdir kisslicer
+    unzip -d kisslicer $src
+  '';
+
+  buildInputs = [
+    makeWrapper
+    unzip
+  ];
+  installPhase = ''
```

Comment:
The convention is to have empty lines between the attributes.

Reply:
and fixed.